src/wallet/rpcwallet.cpp

```diff
@@ -3329,10 +3330,11 @@ static UniValue bumpfee(const JSONRPCRequest& request)
             },
             true, true);
 
-        if (options.exists("confTarget") && options.exists("totalFee")) {
+        auto conf_target_exists = options.exists("confTarget") || options.exists("conf_target");
```

Comment:
There's a type check above that needs to be updated for the alias. (Prefer the alias being a separate commit.)

Reply:
Split alias part into separate commit and added type check.